src/Features/CSharp/Portable/Completion/CompletionProviders/AttributeNamedParameterCompletionProvider.cs

```diff
@@ -151,19 +150,21 @@ private static bool IsAfterNameEqualsArgument(SyntaxToken token)
             return false;
         }
 
-        private static async Task<ImmutableArray<CompletionItem>> GetNameEqualsItemsAsync(
+        private static ImmutableArray<CompletionItem> GetNameEqualsItems(
             CompletionContext context, SemanticModel semanticModel,
             SyntaxToken token, AttributeSyntax attributeSyntax, ISet<string> existingNamedParameters)
         {
             var attributeNamedParameters = GetAttributeNamedParameters(semanticModel, context.Position, attributeSyntax, context.CancellationToken);
             var unspecifiedNamedParameters = attributeNamedParameters.Where(p => !existingNamedParameters.Contains(p.Name));
 
-            var text = await semanticModel.SyntaxTree.GetTextAsync(context.CancellationToken).ConfigureAwait(false);
+            var rightToken = semanticModel.SyntaxTree.FindTokenOnRightOfPosition(context.Position, context.CancellationToken);
+            var shouldAddEquals = !rightToken.IsKind(SyntaxKind.EqualsToken);
```

Comment:
We can define thedisplay suffix here instead of using another conditinoal below. Same for `GetNameColonItems`